src/azul/service/responseobjects/hca_response_v5.py

```diff
@@ -340,6 +345,13 @@ def _merge(self, dict_1, dict_2, identifier):
     def return_response(self):
         return self.apiResponse
 
+    @staticmethod
+    def to_camel_case(s):
+        if not s:
```

Comment:
Is there a case that `s` can be evaluated as `False`? If not, I don't think this defensive code is necessary here. Or, if you are worry about the possible empty `s`, I would go with `assert s` instead.